Backend/Backend/Infrastructure/ComponentDefaults.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Backend.Infrastructure
+{
+    public static class ComponentDefaults
+    {
+        public const string DefaultRoute = "api/v1/[controller]/[action]";
```

Comment:
What `v1` does mean? Maybe it's better to implement routing not like a string variable, but like an entity which is better to change in any time.

Reply:
Maybe it's better to implement routing not like a string variable, but like an entity which is better to change in any time.

Reply:
You can read api versioning paragraph. https://docs.microsoft.com/en-us/aspnet/web-api/overview/web-api-routing-and-actions/attribute-routing-in-web-api-2

Reply:
> Maybe it's better to implement routing not like a string variable, but like an entity which is better to change in any time. For example using builder design pattern.

It would be violation of the KISS principle. Cause Decorator [Route(string)] takes only string, and in your case we should create one more helper to make STRING from your entity.